compiler/src/main/scala/dx/compiler/WorkflowCompiler.scala

```diff
@@ -12,13 +12,16 @@ import scala.collection.immutable.TreeSeqMap
 import dx.translator.DockerRegistry
 import dx.api.DxWorkflow
 import dx.api.DxApplet
+import dx.api.DxProject
 
 case class WorkflowCompiler(extras: Option[Extras],
                             parameterLinkSerializer: ParameterLinkSerializer,
                             useManifests: Boolean,
                             dxApi: DxApi = DxApi.get,
-                            logger: Logger = Logger.get)
-    extends ExecutableCompiler(extras, parameterLinkSerializer, dxApi) {
+                            logger: Logger = Logger.get,
+                            project: DxProject,
+                            folder: String)
+    extends ExecutableCompiler(extras, parameterLinkSerializer, dxApi, project, folder) {
```

Comment:
It looks like project and folder are not used in the superclass, so they do not need to be passed to the superclass constructor. They are also not even used in WorkflowCompiler.

Reply:
I would use it in WorfklowCompiler for native applets in APPS-621 (pending discussion whether to clone).